roxctl/central/debug/resync_check.go

```diff
@@ -0,0 +1,180 @@
+package debug
+
+import (
+	"context"
+	"encoding/json"
+	"os"
+	"path"
+	"sort"
+	"time"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	v1 "github.com/stackrox/rox/generated/api/v1"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/roxctl/common/environment"
+	"github.com/stackrox/rox/roxctl/common/flags"
+	"github.com/stackrox/rox/roxctl/common/util"
+	"google.golang.org/grpc"
+)
+
+const (
+	resyncCheckTimeout = 300 * time.Second
+)
+
+// resyncCheckCommand will outputs alert data before and after reassessing ACS policies.
+func resyncCheckCommand(cliEnvironment environment.Environment) *cobra.Command {
+	var outputDir string
+	var waitFor time.Duration
+
+	c := &cobra.Command{
+		Use:   "resync-check",
+		Short: "Check alerts before and after reassessing policies",
+		Long:  "Check alerts before and after reassessing policies. This should only be used for testing when Secured Clusters have ROX_RESYNC_DISABLED=true",
+		RunE: util.RunENoArgs(func(c *cobra.Command) error {
+			cmd, err := commandWithConnection(cliEnvironment, waitFor, outputDir)
+			if err != nil {
+				return err
+			}
+			before, after, err := cmd.run()
+			if err != nil {
+				return err
+			}
+
+			return cmd.storeFiles(before, after)
+		}),
+	}
+	flags.AddTimeoutWithDefault(c, diagnosticBundleDownloadTimeout)
```

Comment:
I think this should be `resyncCheckTimeout`

```suggestion
	flags.AddTimeoutWithDefault(c, resyncCheckTimeout)
```